OpenEdX/AppDelegate.swift

```diff
@@ -54,6 +61,31 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
+    func application(
+        _ app: UIApplication,
+        open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]
+    ) -> Bool {
+        ApplicationDelegate.shared.application(
+            app,
+            open: url,
+            sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
+            annotation: options[UIApplication.OpenURLOptionsKey.annotation]
+        )
+
+        if GIDSignIn.sharedInstance.handle(url) {
+            return true
+        }
+
+        if MSALPublicClientApplication.handleMSALResponse(
+            url,
+            sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String
+        ) {
+            return true
+        }
```

Comment:
i think we should add check with the config, if a social auth is enabled, then handle the url, e.g.

```
if config.googleConfig.enabled {
        GIDSignIn.sharedInstance.handle(url)
}
```